BAZProjectC1/Billboard/MovieCell.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  MovieCell.swift
+//  BAZProjectC1
+//
+//  Created by efloresco on 21/09/22.
+//
+
+import UIKit
+
+class MovieCell: UICollectionViewCell {
+    @IBOutlet weak var imgDetail: UIImageView!
+    @IBOutlet weak var lblTitle: UILabel!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    }
+    
+    func configureCellWithUrl(movieInfo: Movie) {
```

Comment:
Creo que el nombre de esta función no refleja lo que realmente está haciendo.
Podrías renombrarla de esta forma:
```suggestion
    func configureCell(with movie: Movie) {
```

Reply:
resuelto